packages/ui/components/input/NumberInput.tsx

```diff
@@ -54,73 +52,59 @@ export const NumberInput = <FieldValues, FieldName extends Path<FieldValues>>({
     {}
   )
 
-  const sharedProps = {
```

Comment:
Looks like there's more changes here than just `onPlus` and `onMinus`. Did this refactor change the view functionality at all or how it looks?

Reply:
Ah yeah, I should have explained. We had unnecessarily duplicated code, if there were plus/minus functions provided or not. I decided it was fine to use the same container-wrapped input if there are no plus/minus buttons since it looks the same and makes the code more readable.